packages/wallet-core/src/store/types.ts

```diff
@@ -202,6 +203,7 @@ export type HistoryItem = NFTSendHistoryItem | SendHistoryItem | SwapHistoryItem
 
 export enum ExperimentType {
   ManageAccounts = 'manageAccounts',
+  ReportErrors = 'reportErrors',
```

Comment:
Why does this need to be an experiement? Seems pretty straightforward

Reply:
experiements are for when you need to enable something in the experiements tab in the wallet

Reply:
But this feature is necessary and we want to it work at all times
If a user misses an error in the log, makes everyone's life more difficult